app/reducers/meetings.ts

```diff
@@ -6,6 +6,7 @@ import createEntityReducer from 'app/utils/createEntityReducer';
 import joinReducers from 'app/utils/joinReducers';
 import { Meeting } from '../actions/ActionTypes';
 import { mutateReactions } from './reactions';
+import type { Moment } from 'moment-timezone';
```

Comment:
Very nitpicky: But import this right under the `moment` import on line 1?

Reply:
We have specifically configured this to not be allowed. Type imports come last